shadows/framework/src/main/java/org/robolectric/shadows/ShadowSensorManager.java

```diff
@@ -50,9 +50,19 @@ public void addSensor(int sensorType, Sensor sensor) {
 
   /** Adds a {@link Sensor} to the {@link SensorManager} */
   public void addSensor(Sensor sensor) {
+    if (sensor == null) {
+      return;
```

Comment:
nit: can you add a `Preconditions.checkNotNull(sensor)` check for not null? I don't see a use case for a null sensor. Same below as well.

Reply:
nvm I pushed that change :)

Reply:
Thanks, @hoisie . I'm sorry that I don't reply promptly, because I'm sleeping.